# TaskManagerStateMsg.msg

bool start 
int16 taskId 
int16 toolToReach
bool toolReached
bool finished
bool stopTime
int16 imageId